src/appleseed.studio/main/main.cpp

```diff
@@ -271,16 +271,16 @@ namespace
         }
     }
 
-    QtMsgHandler g_previous_message_handler = nullptr;
+    QtMessageHandler g_previous_message_handler = nullptr;
 
-    void message_handler(QtMsgType type, const char* msg)
+    void message_handler(QtMsgType type, const QMessageLogContext& context, const QString& msg)
     {
 #ifdef __APPLE__
         // Under certain circumstances (under an macOS virtual machine?), a bogus warning
         // message is repeatedly printed to the console. Disable this warning message.
         // See https://github.com/appleseedhq/appleseed/issues/254 for details.
         if (type == QtWarningMsg &&
-            strcmp(msg, "QCocoaView handleTabletEvent: This tablet device is unknown (received no proximity event for it). Discarding event.") == 0)
+            msg == "QCocoaView handleTabletEvent: This tablet device is unknown (received no proximity event for it). Discarding event.")
```

Comment:
I wonder if this kludge is still necessary. I'll check once I had it running on a macOS VM.